packages/mds-utils/exceptions/exception-messages.ts

```diff
@@ -0,0 +1,2 @@
+export const INITIALIZE_CLIENT_MESSAGE =
+  'Client is not connected! Have you tried initializing the client with .initialize()?'
```

Comment:
I think this should be exported as an object to get some better intellisense and allow for simpler expansion.
```ts
export const ExceptionMessages = {
  INITIALIZE_CLIENT_MESSAGE: 'Client is not connected! Have you tried initializing the client with .initialize()?'
}
```